Input/input.html

```diff
@@ -0,0 +1,176 @@
+              Modal
+              </a>
+          </li>
+          <li>
+              <a href="../Rating/rating.html" class="sidebar-menu" rel="noopener noreferrer">
+              Rating
+              </a>
+          </li>
+          <li>
+              <a href="../Toast/toast.html" class="sidebar-menu" rel="noopener noreferrer">
+              Toast
+              </a>
+          </li>
+          <li>
+              <a href="../SimplifiedGrid/grid.html" class="sidebar-menu" rel="noopener noreferrer">
+              Simplified Grid
+              </a>
+          </li>
+      </ul>
+      </div>
+      <div class="main">
+        <h1 class="font-xl margin-m2">Input</h1>
+        <br />
+        <hr />
+        <h3 class="font-medium margin-m2">Primary Inputs With Textbox</h3>
+        <p  class="font-medium label-text"> Primary input with textbox is one way of taking input from the user.</p>
+
+        <div class="box container-flex flex-column">
+          <textarea
+            class="text-area"
+            cols="300"
+            rows="10"
+            placeholder="Enter a text"
+          >
+          </textarea>
+          <br />
+          <br />
+          <input type="text" class="input" placeholder="Enter text" /> <br />
+          <br />
+          <input type="number" class="input" placeholder="Enter Number" />
+          <br />
+          <br />
+          <input type="password" class="input" placeholder="Enter password" />
+          <br />
+          <br />
+        </div>
+        <div class="margin-m2 padding-p2">
+          <code class="code-container container-flex flex-center">
+            <iframe src="https://carbon.now.sh/embed/kgvYDso4jyCZOGvOihlf" style="width: 620px; height: 167px; border:0; transform: scale(1); overflow:hidden;"
+             sandbox="allow-scripts allow-same-origin">
+            </iframe>
+          </code>
+        </div>
+        <h3 class="font-medium margin-m2">Inputs with Validation Error Style </h3>
+        <p class="font-medium label-text">These inputsare validated first and then give any kind of access to the user.</p>
+        <div class="box">
+          <label for="demo" class="font-medium label-text">Email</label> <br />
+          <input type="email" name="demo" class="input" /> <br />
+          <label for="demo" class="font-medium label-text">Password</label>
+          <br />
+          <input
+            type="password"
+            class="input"
+            value="I@ampassword"
+            pattern="^.{8,}$"
+            required
+            placeholder="Enter Password"
+          />
+        </div>
+        <div class="margin-m2 padding-p2">
+          <code class="code-container container-flex flex-center">
+            <iframe
+            src="https://carbon.now.sh/embed/7KjUFXucufD7u8x6HQs1"
+            style="width: 620px; height: 167px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
you can give this style seprately in css file instead of inline.

Reply:
i dont know for iframes but i will study it.